plugins/module_utils/linode_helper.py

```diff
@@ -128,7 +131,15 @@ def handle_updates(
 
         old_value = parse_linode_types(getattr(obj, key))
 
-        if new_value != old_value:
+        has_diff = new_value != old_value
+
+        # We should convert properties to sets
+        # if they are annotated as unordered in the
+        # Python SDK.
+        if property_metadata.get(key).unordered:
+            has_diff = set(old_value) != set(new_value)
```

Comment:
Apparently the API request function doesn't like JSON serializing sets so I couldn't mutate old_value and new_value here